packages/terra-framework-site/src/app/components/Components.jsx

```diff
@@ -1,14 +1,35 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { Route, Switch, Redirect, withRouter } from 'react-router-dom';
 import { siteConfigPropType } from '../../componentConfig';
 
 const propTypes = {
   config: siteConfigPropType,
-  location: React.PropTypes.shape({
-    pathname: React.PropTypes.string.isRequired,
+  location: PropTypes.shape({
+    pathname: PropTypes.string.isRequired,
   }),
 };
 
+const generateRoutes = config => (
+  Object.keys(config).map((componentKey) => {
+    const example = config[componentKey].example;
+    if (example) {
+      return (
+        <Route key={example.path} path={example.path} component={example.component} />
+      );
+    }
+    return undefined;
+  })
+);
+
+const generateRedirect = (config) => {
```

Comment:
On line 25, we use parenthesis but on line 13, we don't use parenthesis around the single parameter. Do we have a styling rule regarding this? Consistent styling throughout is important.

Reply:
I feel you. This does seem like an arbitrary difference, but these styles around arrow functions are built into our lint rules: https://github.com/airbnb/javascript#arrows--implicit-return

Basically, if an arrow function has a single parameter and the function body is a single returned expression (and therefore wrapped in '()' vs. '{}'), they prefer the parentheses around the parameter to be removed. If there's more than one parameter, the parentheses come back. If the function body contains more than a explicit return statement, parentheses come back, regardless of the parameter count. 

:man_shrugging: